app/controllers/system/admin/instance_announcements_controller.rb

```diff
@@ -32,8 +32,10 @@ def update
   end
 
   def destroy
-    redirect_to admin_announcements_path,
-                success: t('.success', title: @announcement.title) if @announcement.destroy
+    if @announcement.destroy
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.